### PR TITLE
Improve run command exception handling

### PR DESCRIPTION
**Pull request by MHeus**
Fixes unhandled timeout exceptions and exits when invalid json is received

### CODE REVIEW

File: src/doing/utils.py [hunk 1]
```diff
@@ -223,16 +223,20 @@ def run_command(command: str, allow_verbose=True):
...
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+            encoding=encoding,
+            shell=True,
+            timeout=15,
+        )
+    except Exception as e:
+        console.print(e)
```
Comment (timvink) [inline src/doing/utils.py:238]: Before print the error, I'd also like to see the encoding used and the command trying to run. Something like:

```python
console.print(f"There was an error. Ran the following command with encoding '{encoding}':")
console.print(f"[bright_black]{command}[/bright_black]")
console.print(f"[dark_orange3]{e}[/dark_orange3]")
```

**Review by timvink — CHANGES_REQUESTED** [review]

File: src/doing/utils.py (hunk 1 above)
Comment (MHeus) [reply src/doing/utils.py:238]: Fixed by only catching timeout

**Comment by timvink** [comment]
Thanks again!

**Comment by timvink** [comment]
Released in https://github.com/ing-bank/doing-cli/releases/tag/v1.2.7

**Pull request merged**